src/pages/Reports/components/EventsInputs/EventsInputs.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState, forwardRef, useImperativeHandle } from 'react';
+import { useTranslation } from 'react-i18next';
+import UserStore from '../../../../stores/User.store';
+import NodeGroupSelect from '../../../../common/NodeGroupSelect/NodeGroupSelect';
+import UnitSelect from '../../../../common/UnitSelect/UnitSelect';
+import LabeledInput from '../LabeledInput/LabeledInput';
+import useStyles from '../Input/Input.styles.js';
+
+const EventsInputs = ({
+  unitsOptionalRoles,
+  nodeGroupsRequiredRoles,
+  nodeGroupsOptionalRoles,
+  setCanSubmit,
+  setDataToSend,
+ }, ref) => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+  const currentUser = UserStore.userProfile;
+  const [selectedNodeGroups, setSelectedNodeGroups] = useState([]);
+  const [selectedUnits, setSelectedUnits] = useState([]);
+
+  useEffect(() => {
+    setCanSubmit(
+      !(nodeGroupsRequiredRoles.includes(currentUser.role) && !selectedNodeGroups.length),
```

Comment:
Change to a more intuitive syntax
```suggestion
      !nodeGroupsRequiredRoles.includes(currentUser.role) || selectedNodeGroups.length,
```

Reply:
it is more readable that way